src/chrome/chromeDebugAdapter.ts

```diff
@@ -1405,7 +1405,7 @@ export abstract class ChromeDebugAdapter implements IDebugAdapter {
         this.reportBpTelemetry(args);
         if (args.source.path) {
             args.source.path = this.displayPathToRealPath(args.source.path);
-            args.source.path = utils.fixDriveLetterAndSlashes(args.source.path);
```

Comment:
I am removing fixDriveLetterAndSlashes() since that is called internally by canonicalizeUrl()